### ISSUE TITLE
Bad import in python client if using type type format

### ISSUE THREAD

**Issue by zaibon** [Jumpscale/go-raml#266]
Example raml type definition:

```raml
SystemStatus:
    properties:
      online:
        type: boolean
        description: Always present and true if service is online
        required: false
      clock:
        type: integer
        format: int64
        description: System clock of server in milliseconds since epoch
        required: false
      version:
        type: string
        description: ZeroTier Central server version
        required: false
      uptime:
        type: integer
        format: int64
        description: 'Server uptime in milliseconds (service, not host)'
        required: false
      user:
        description: Current user or null if not logged in
        type: User
        required: false
      stripePublishableKey:
        type: string
        description: Stripe API publishable key for payment info entry or null if payments not enabled on this server or if not logged in
        required: false
      paidPlans:
        type: object
        description: Paid plan info for web UI or null if payment not enabled on this server
        required: false
      smsEnabled:
        type: boolean
        description: True if this server can send SMS messages
        required: false
      loginMethods:
        type: object
        description: Login methods supported by this server
        required: false
```

Snippet from the generated code :
`from .int64 import int64`

**Comment by iwanbk** [comment]
duplicate of #259, fixed in #262

**Comment by iwanbk** [comment]
fixed in #262
